### PR TITLE
LPD-43220 Apply ERC to Categories Filter Widget (Staging PR)

### PR DESCRIPTION
**Pull request by jkappler**
Apply ERC to Categories Filter Widget (Staging PR)

# What is this trying to solve?

[Link to ticket](https://issues.liferay.com/browse/LPD-43247)

We are working on replacing the internal ids for external reference codes in categories filter, this PR deals with the staging part of it

# How am I fixing it?

There has been a significant change in storing the preferences for the current site. We just store them without group, so that they are found when importing the site

# How can you verify that it works?

Run the provided tests

### CODE REVIEW

**Comment by jkappler** [comment]
ci:test:sf

File: modules/apps/asset/asset-categories-navigation-web/src/main/java/com/liferay/asset/categories/navigation/web/internal/display/context/AssetCategoriesNavigationDisplayContext.java [hunk 1]
```diff
@@ -285,7 +294,36 @@ private String[] _getAssetVocabularyIds() {
...
+		PortletPreferences portletPreferences) {
+
+		List<Long> assetVocabularyIds = new ArrayList<>();
+
+		String[] assetVocabularyExternalReferenceCodes =
+			GetterUtil.getStringValues(
+				portletPreferences.getValues(
+					"assetVocabularyExternalReferenceCodes", null));
+
+		for (String assetVocabularyGroupExternalReferenceCode :
```
Comment (adolfopa) [inline modules/apps/asset/asset-categories-navigation-web/src/main/java/com/liferay/asset/categories/navigation/web/internal/display/context/AssetCategoriesNavigationDisplayContext.java:310]: Shouldn't this variable be named `assetVocabularyExternalReferenceCode`? (without the "Group").

Comment (jkappler) [reply modules/apps/asset/asset-categories-navigation-web/src/main/java/com/liferay/asset/categories/navigation/web/internal/display/context/AssetCategoriesNavigationDisplayContext.java]: You are right, thanks @adolfopa! I changed it

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Comment by adolfopa** [comment]
ci:stop

**Comment by adolfopa** [comment]
ci:test:stable

**Comment by Mikellorza** [comment]
ci:forward

**Pull request closed without merging**